sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_client.py

```diff
@@ -115,7 +115,7 @@ def close(self):
 
         :return: None
         """
-        for handler in self._handlers:
+        for handler in set(self._handlers):
```

Comment:
are there instances in which Sender/Receiver can be added multiple times to the same client?

Reply:
according to our current design, nope, the handlers are maintained internally only and users could not add existing handler to the client.

Reply:
this is actually doing a copy of the handler array, otherwise the handler array would be changing while performing the `handler.close()` operation as the handler.close method would remove itself from the handlers of service bus client.